zentral/contrib/osquery/api_views.py

```diff
@@ -25,6 +25,11 @@
 from .tasks import export_distributed_query_results
 
 
+class AutomaticTableConstructionFilter(filters.FilterSet):
+    configuration_id = filters.NumberFilter(field_name='configuration__id')
```

Comment:
Probably better to user a [ModelChoiceFilter](https://github.com/zentralopensource/zentral/blob/12ae493bfc831ab024c42e447bafec432b51521a/zentral/contrib/santa/api_views.py#L170). Also maybe test if the config doesn't exist, the kind of error you are getting.